apps/system/js/hierarchy_manager.js

```diff
@@ -58,6 +58,7 @@
      * @type {Array}
      */
     PRIORITIES: [
+      'OverlayWindowManager',
```

Comment:
Thinking about this twice. I still believe that BuddyUp should use another approach if the screen reader is active. There is no point about showing visual feedback to a blind user. But still the screen reader should also be able to read the GOWM. That doesn't hurt and it would help future use cases different to the BuddyUp one.